spec/integration/ldap_spec.rb

```diff
@@ -64,38 +65,39 @@
         :owner => 'uid=quentin,ou=people,dc=example,dc=org',
         :uniquemember => ['uid=samuel', 'uid=uma', 'uid=john']
       }
-      Hydra::LDAP.create_group('PulpFiction', attrs).should be_true
+      expect(Hydra::LDAP.create_group('PulpFiction', attrs)).to be true
     end
 
     after do
-      Hydra::LDAP.delete_group('PulpFiction').should be_true
+      Hydra::LDAP.delete_group('PulpFiction')
     end
 
     it "should return a list of groups owned by quentin" do
       attrs = {
-        :cn => 'TR',
+        :cn => 'TrueRomance',
         :objectclass => 'groupofuniquenames',
         :description => 'True Romance is another movie by Q',
         :owner => 'uid=quentin,ou=people,dc=example,dc=org',
         :uniquemember => ['uid=christian', 'uid=patricia', 'uid=dennis']
       }
-      Hydra::LDAP.create_group('TrueRomance', attrs).should be_true
+      expect(Hydra::LDAP.create_group('TrueRomance', attrs)).to be true
       filter = Net::LDAP::Filter.construct("(owner=uid=quentin,ou=people,dc=example,dc=org)")
 
-      Hydra::LDAP.groups_owned_by_user(filter, ['owner', 'cn']){ |result| result.map{ |r| r[:cn].first } }.should == ['PulpFiction', 'TrueRomance']
+      groups = Hydra::LDAP.groups_owned_by_user(filter, ['owner', 'cn']){ |result| result.map{ |r| r[:cn].first } }
+      expect((groups - ['PulpFiction', 'TrueRomance'])).to be_empty
```

Comment:
More idiomatic to do something like `expect(groups).to contain_exactly(['PulpFiction', 'TrueRomance'])`